pkg/utils/pod/scheduling.go

```diff
@@ -84,8 +83,7 @@ func HasDoNotDisrupt(pod *v1.Pod) bool {
 	if pod.Annotations == nil {
 		return false
 	}
-	return pod.Annotations[v1alpha5.DoNotEvictPodAnnotationKey] == "true" ||
-		pod.Annotations[v1beta1.DoNotDisruptAnnotationKey] == "true"
+	return pod.Annotations[v1beta1.DoNotDisruptAnnotationKey] == "true"
```

Comment:
We can't drop this yet. This is needed for the backwards compat that we promised for this deprecated annotation up until v1. Should add a `TODO` to drop this in v1